lottie/src/main/java/com/airbnb/lottie/LottieCompositionFactory.java

```diff
@@ -106,14 +104,13 @@ public static void setMaxCacheSize(int size) {
    * The asset file name will be used as a cache key so future usages won't have to parse the json again.
    * However, if your animation has images, you may package the json and images as a single flattened zip file in assets.
    *
-   * @see #fromZipStreamSync(ZipInputStream, String)
    */
   @WorkerThread
   public static LottieResult<LottieComposition> fromAssetSync(Context context, String fileName) {
     try {
       String cacheKey = "asset_" + fileName;
       if (fileName.endsWith(".zip")) {
-        return fromZipStreamSync(new ZipInputStream(context.getAssets().open(fileName)), cacheKey);
+        // return fromZipStreamSync(new ZipInputStream(context.getAssets().open(fileName)), cacheKey);
```

Comment:
Cette ligne ne devrait-elle pas plutôt être ceci : 

```suggestion
        return null;
```